code/M3D/m3d.h

```diff
@@ -70,11 +70,14 @@ typedef double M3D_FLOAT;
 #endif
 #if !defined(M3D_SMALLINDEX)
 typedef uint32_t M3D_INDEX;
+#define M3D_UNDEF 0xffffffff
 #define M3D_INDEXMAX 0xfffffffe
 #else
 typedef uint16_t M3D_INDEX;
+#define M3D_UNDEF 0xffff
 #define M3D_INDEXMAX 0xfffe
 #endif
+#define M3D_NOTDEFINED 0xffffffff
```

Comment:
Where is the differece between the tw others?

Reply:
Should you have copied a few more lines from the code and you'd see the ifdef which defines the difference. You can configure the index size with a define.

Reply:
Ok, thanks for the update.